### PR TITLE
base: remove dirseq dependency from networkmanager

### PR DESCRIPTION
**Pull request by webispy**
For future refactoring, remove the directive sequencer dependency from
the network manager module.

Signed-off-by: Inho Oh <inho.oh@sk.com>

### CODE REVIEW

File: src/base/nugu_network_manager.c [hunk 1]
```diff
@@ -106,14 +105,27 @@ struct _nugu_network {
...
+	void *attachment_callback_userdata;
 };
 
 typedef struct _nugu_network NetworkManager;
 
 static void on_directive(enum nugu_equeue_type type, void *data, void *userdata)
 {
+	NetworkManager *nm = userdata;
+
+	if (nm->directive_callback == NULL)
```
Comment (kimhyungrok) [inline src/base/nugu_network_manager.c:124]: Is it guaranteed than nm is not NULL? (It's same in 142 line below.)

**Comment by kimhyungrok** [comment]
Typo error in commit message. "... sequcne...".

**Comment by webispy** [comment]
> Typo error in commit message. "... sequcne...".

Thank you :)

File: src/base/nugu_network_manager.c (hunk 1 above)
Comment (webispy) [reply src/base/nugu_network_manager.c:124]: Yes. userdata is always not NULL.

Line 636:

	nm = calloc(1, sizeof(NetworkManager));
	if (!nm) {
		nugu_error_nomem();
		return NULL;
	}

	/* Received message from server */
	nugu_equeue_set_handler(NUGU_EQUEUE_TYPE_NEW_DIRECTIVE, on_directive,
				on_destroy_directive, nm);
	nugu_equeue_set_handler(NUGU_EQUEUE_TYPE_NEW_ATTACHMENT, on_attachment,
				on_destroy_attachment, nm);

Comment (kimhyungrok) [reply src/base/nugu_network_manager.c:124]: I understand. Thanks.

**Pull request merged**